JTCalendar/JTCalendarDayView.m

```diff
@@ -199,17 +199,24 @@ - (void)setSelected:(BOOL)selected animated:(BOOL)animated
     circleView.transform = CGAffineTransformIdentity;
     CGAffineTransform tr = CGAffineTransformIdentity;
     CGFloat opacity = 1.;
+	JTCalendarAppearance *calendarAppearance = self.calendarManager.calendarAppearance;
```

Comment:
I made this modifications to somewhat cache the `calendarAppearance` but also reduce the verbosity below.